docs/source/getting-started.rst

```diff
@@ -59,3 +59,29 @@ Below is a simple example:
 
 Full example code is available `here <https://github.com/EthWorks/useDapp/tree/master/packages/example>`_.
 
+Creating new hook
+------------
+
+Creating a custom hook with use of our core hooks is very easy, as example let's write a *useEtherBalance* hook:
+
+.. code-block:: javascript
+
+  export function useEtherBalance(address: string | Falsy) {
+    const multicallAddress = useMulticallAddress() // gets address of multicall contract
+    const getEthBalanceCall = address &&
+      MultiCallABI.encodeFunctionData('getEthBalance', [address])
```

Comment:
It is super non-obvious why we use getEthBalance - need explanation.

Reply:
Also a paragraph on what the hell is multicall is a good idea ;)

Reply:
We moved to useTokenBalance and it's not relevant anymore